docs/sql/index.md

```diff
@@ -9,14 +9,18 @@ has_children: true
 
 Open Distro for Elasticsearch SQL lets you write queries in SQL rather than the [Elasticsearch query domain-specific language (DSL)](../elasticsearch/full-text). If you're already familiar with SQL and don't want to learn the query DSL, this feature is a great option.
 
-To use the feature, send requests to the `_opendistro/_sql` URI. You can use a request parameter or the request body (recommended).
+SQL UI is now supported. Use the SQL UI to easily run on-demand SQL queries, translate SQL into its REST equivalent, and view and save results as text, JSON, JDBC, or CSV.
+
+![Kibana SQL UI plugin](../images/sql.png)
+
+To use the REST API, send requests to the `_opendistro/_sql` URI. You can use a request parameter or the request body (recommended).
 
 ```sql
-GET https://<host>:<port>/_opendistro/_sql?sql=select * from my-index limit 50
+POST _opendistro/_sql?sql=select * from my-index limit 50
 ```
```

Comment:
Since `GET` endpoint is removed already due to security concern in https://github.com/opendistro-for-elasticsearch/sql/pull/414, could you help remove it from the documentation?

Reply:
Changed. Please review.